vars.tf

```diff
@@ -48,6 +48,16 @@ variable "docker_tag" {
   default = "latest"
 }
 
+variable "dynamo_access_key_id" {
+  description = "AWS credential for DynamoDB permissions for sildisco to log user logins"
```

Comment:
Where is this key going to be created? Would it make sense to manage it in this workspace?

Reply:
I hadn't thought that far ahead.  If you think that's the best solution, I'll try to include that.

Reply:
I can't think of a better place. That would save the manual step of copying a key from elsewhere into this variable.